freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/FreeIpaUsersStateProvider.java

```diff
@@ -24,7 +24,7 @@
     // TODO add other Cloudera-managed groups?
     // TODO handle name conflicts between ipa and ums? e.g., should ums "admins" be ipa "admins"?
     @VisibleForTesting
-    static final List<String> IPA_ONLY_GROUPS = List.of("admins", "editors", "ipausers", "trust admins");
+    static final List<String> IPA_ONLY_GROUPS = List.of("editors", "ipausers", "trust admins");
```

Comment:
I'm guessing that you removed it here so that the FMS can remove users from the "admins" group. 

The "admins" group may need special treatment. We don't want to remove the admins group altogether or else the FMS won't be able to use the admin user to sync in the future. Perhaps we need finer-grained whitelisting, in which case it may make sense to move this to the state difference calculation. i.e., we want to remove group membership to `admins` (except for the `admin` user) but we don't want to delete the `admins` group

Reply:
Changed